### ISSUE TITLE
PFS8 node0 transfer to node3 expires

### ISSUE THREAD

**Issue by agatsoh** [raiden-network/light-client#2443]
## Steps to Reproduce
1. 
2. 

## Expected Result
- 

## Actual Result
- 
```
{
  "task": "<TransferTask: {'from': 0, 'to': 3, 'amount': 1000000000000000, 'expected_http_status': 200, 'identifier': 154894145181000002}>",
  "event": "Task errored",
  "logger": "scenario_player.tasks.base",
  "level": "error",
  "greenlet_name": "orchestration",
  "timestamp": "2020-12-10 00:30:43.392009",
  "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/base.py\", line 140, in __call__\n    return_val = self._run(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/api_base.py\", line 59, in _run\n    f'HTTP status code \"{resp.status_code}\" while fetching {url}. '\nscenario_player.exceptions.RESTAPIStatusMismatchError: HTTP status code \"409\" while fetching http://127.0.0.1:37807/api/v1/payments/0x59105441977ecD9d805A4f5b060E34676F50F806/0x497A9851e236F051FC07EaaEe3f0334Ee85979df. Expected 200: Transfer expired."
}
```

Fails at:
`TransferTask: {'from': 0, 'to': 3, 'amount': 1000000000000000, 'expected_http_status': 200, 'identifier': 154894145181000002}`

**Transfer expired.**

```
Traceback (most recent call last):
File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/base.py\", line 140, in __call__
return_val = self._run(*args, **kwargs)
File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/api_base.py\", line 59, in _run
f'HTTP status code \"{resp.status_code}\" while fetching {url}. '
scenario_player.exceptions.RESTAPIStatusMismatchError: HTTP status code \"409\" while fetching http://127.0.0.1:37807/api/v1/payments/0x59105441977ecD9d805A4f5b060E34676F50F806/0x497A9851e236F051FC07EaaEe3f0334Ee85979df. Expected 200: Transfer expired.
```

node0 till the end apparently could not setup a webrtc channel
```
2020-12-10T00:30:43.441Z [info] Couldn't set up WebRTC dataChannel, retrying 0x3Deb22082512004757846980a573CA8C7146e261|0x497A9851e236F051FC07EaaEe3f0334Ee85979df RTC: dataChannel closed
2020-12-10T00:30:43.449Z [info] %c action     color: #03A9F4; font-weight: bold {
  type: 'rtc/channel',
  payload: undefined,
  meta: { address: '0x497A9851e236F051FC07EaaEe3f0334Ee85979df' }
}
```

LockedTransfer message

```
2020-12-10T00:03:20.770Z [info] %c action     color: #03A9F4; font-weight: bold {
  type: 'transfer/signed',
  payload: {
    message: {
      type: 'LockedTransfer',
      message_identifier: BigNumber { _hex: '0x017649f59839', _isBigNumber: true },
      chain_id: BigNumber { _hex: '0x05', _isBigNumber: true },
      token_network_address: '0x3EA2a1fED7FdEf300DA19E97092Ce8FdF8bf66A3',
      channel_identifier: BigNumber { _hex: '0x40e6', _isBigNumber: true },
      nonce: BigNumber { _hex: '0x03', _isBigNumber: true },
      transferred_amount: BigNumber { _hex: '0x038d7ea4c68000', _isBigNumber: true },
      locked_amount: BigNumber { _hex: '0x038d7ea4c68000', _isBigNumber: true },
      locksroot: '0x6aa8eb78e37402b633bd5a8bdb911e5847d414a133103566f19b5ed8fbdc1258',
      payment_identifier: BigNumber { _hex: '0x02264b67854d2140', _isBigNumber: true },
      token: '0x59105441977ecD9d805A4f5b060E34676F50F806',
      recipient: '0x28c91e1aaff0657E17C5D917bA483C224DDD67d7',
      lock: {
        amount: [BigNumber],
        expiration: [BigNumber],
        secrethash: '0xdc74a5fde42db3bd5af146aa28eefc64b16dc347898f2597cd19dd7427f1c0e5'
      },
      target: '0x497A9851e236F051FC07EaaEe3f0334Ee85979df',
      initiator: '0x3Deb22082512004757846980a573CA8C7146e261',
      metadata: { routes: [Array] },
      signature: '0x413f0b0755b6399c3d52599f4d2ee65b39f8ef448584ca37bf14551ded563f3d6a92c69ae33ffe0dfae2ca4f8e56e6f8c1904405da5d8fb852b1304045d113df1b'
    },
    fee: BigNumber { _hex: '0x00', _isBigNumber: true },
    partner: '0x28c91e1aaff0657E17C5D917bA483C224DDD67d7'
  },
  meta: {
    secrethash: '0xdc74a5fde42db3bd5af146aa28eefc64b16dc347898f2597cd19dd7427f1c0e5',
    direction: 'sent'
  }
}
```

We the transferExpire request action
**transfer/expire/request**
```
2020-12-10T00:30:43.376Z [info] %c action     color: #03A9F4; font-weight: bold {
  type: 'transfer/expire/request',
  meta: {
    secrethash: '0xdc74a5fde42db3bd5af146aa28eefc64b16dc347898f2597cd19dd7427f1c0e5',
    direction: 'sent'
  }
}
```

**transfer/failure** action
```
2020-12-10T00:30:43.377Z [info] %c action     color: #03A9F4; font-weight: bold {
  type: 'transfer/failure',
  payload: RaidenError: Transfer expired.
      at MergeMapSubscriber.project (/lc/index.js:250113:46)
      at MergeMapSubscriber.module.exports.__webpack_modules__.42713.MergeMapSubscriber._tryNext (/lc/index.js:204027:27)
      at MergeMapSubscriber.module.exports.__webpack_modules__.42713.MergeMapSubscriber._next (/lc/index.js:204017:18)
      at MergeMapSubscriber.module.exports.__webpack_modules__.65183.Subscriber.next (/lc/index.js:198786:18)
      at Observable._subscribe (/lc/index.js:209058:20)
      at Observable._trySubscribe (/lc/index.js:198183:25)
      at Observable.subscribe (/lc/index.js:198169:22)
      at MergeMapOperator.module.exports.__webpack_modules__.42713.MergeMapOperator.call (/lc/index.js:203997:23)
      at Observable.subscribe (/lc/index.js:198164:31)
      at Object.innerSubscribe (/lc/index.js:199250:23) {
    details: { block: '3899169' }
  },
  meta: {
    secrethash: '0xdc74a5fde42db3bd5af146aa28eefc64b16dc347898f2597cd19dd7427f1c0e5',
    direction: 'sent'
  },
  error: true
}
```

**transfer/expire/success**
```
2020-12-10T00:30:43.392Z [info] %c action     color: #03A9F4; font-weight: bold {
  type: 'transfer/expire/success',
  payload: {
    message: {
      type: 'LockExpired',
      message_identifier: BigNumber { _hex: '0x01764a0ea8b8', _isBigNumber: true },
      chain_id: BigNumber { _hex: '0x05', _isBigNumber: true },
      token_network_address: '0x3EA2a1fED7FdEf300DA19E97092Ce8FdF8bf66A3',
      channel_identifier: BigNumber { _hex: '0x40e6', _isBigNumber: true },
      nonce: BigNumber { _hex: '0x04', _isBigNumber: true },
      transferred_amount: BigNumber { _hex: '0x038d7ea4c68000', _isBigNumber: true },
      locked_amount: BigNumber { _hex: '0x00', _isBigNumber: true },
      locksroot: '0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470',
      recipient: '0x28c91e1aaff0657E17C5D917bA483C224DDD67d7',
      secrethash: '0xdc74a5fde42db3bd5af146aa28eefc64b16dc347898f2597cd19dd7427f1c0e5',
      signature: '0x51dfc365086bc7bef72a0bb646f320dc185e6509353d313c05f2b6b3e234403f6d45137c9e2e7cc331dd83713b3b2e6cef098a5e35189635b25b63230f4fe5e31c'
    },
    partner: '0x28c91e1aaff0657E17C5D917bA483C224DDD67d7'
  },
  meta: {
    secrethash: '0xdc74a5fde42db3bd5af146aa28eefc64b16dc347898f2597cd19dd7427f1c0e5',
    direction: 'sent'
  }
}
```

**message/send/request**
```
2020-12-10T00:30:43.417Z [info] %c action     color: #03A9F4; font-weight: bold {
  type: 'message/send/request',
  payload: {
    message: {
      type: 'LockExpired',
      message_identifier: BigNumber { _hex: '0x01764a0ea8b8', _isBigNumber: true },
      chain_id: BigNumber { _hex: '0x05', _isBigNumber: true },
      token_network_address: '0x3EA2a1fED7FdEf300DA19E97092Ce8FdF8bf66A3',
      channel_identifier: BigNumber { _hex: '0x40e6', _isBigNumber: true },
      nonce: BigNumber { _hex: '0x04', _isBigNumber: true },
      transferred_amount: BigNumber { _hex: '0x038d7ea4c68000', _isBigNumber: true },
      locked_amount: BigNumber { _hex: '0x00', _isBigNumber: true },
      locksroot: '0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470',
      recipient: '0x28c91e1aaff0657E17C5D917bA483C224DDD67d7',
      secrethash: '0xdc74a5fde42db3bd5af146aa28eefc64b16dc347898f2597cd19dd7427f1c0e5',
      signature: '0x51dfc365086bc7bef72a0bb646f320dc185e6509353d313c05f2b6b3e234403f6d45137c9e2e7cc331dd83713b3b2e6cef098a5e35189635b25b63230f4fe5e31c'
    }
  },
  meta: {
    address: '0x28c91e1aaff0657E17C5D917bA483C224DDD67d7',
    msgId: '1607560243384'
  }
}
```

[scenario-player-run_pfs8_mediator_goes_offline_2020-12-09T23:54:11.log](https://github.com/raiden-network/light-client/files/5671243/scenario-player-run_pfs8_mediator_goes_offline_2020-12-09T23.54.11.log)

[pfs8_node0_node3_10Dec2020.tar.gz](https://github.com/raiden-network/light-client/files/5671245/pfs8_node0_node3_10Dec2020.tar.gz)

**Comment by agatsoh** [comment]
This was observed again for BF1 11th Dec 2020
[scenario-player-run_bf1_basic_functionality_2020-12-10T21:17:12.log](https://github.com/raiden-network/light-client/files/5678476/scenario-player-run_bf1_basic_functionality_2020-12-10T21.17.12.log)


[bf1_node0_node3_11thDec2020.tar.gz](https://github.com/raiden-network/light-client/files/5678478/bf1_node0_node3_11thDec2020.tar.gz)

**Comment by christianbrb** [comment]
Going to get fixed by #2453

**Comment by agatsoh** [comment]
This occurred today on two scenarios attaching logs here.
BF7
[scenario-player-run_bf7_long_path_2020-12-22T00:22:27.log](https://github.com/raiden-network/light-client/files/5729377/scenario-player-run_bf7_long_path_2020-12-22T00.22.27.log)


[bf7_node0_node7_22Dec2020.tar.gz](https://github.com/raiden-network/light-client/files/5729374/bf7_node0_node7_22Dec2020.tar.gz)

BF6

**Comment by agatsoh** [comment]
29th December 2020 this failure was seen for 

BF4

[scenario-player-run_bf4_multi_payments_same_node_2020-12-29T00:28:28.log](https://github.com/raiden-network/light-client/files/5750343/scenario-player-run_bf4_multi_payments_same_node_2020-12-29T00.28.28.log)
[bf4_node0_node3_29Dec2020.tar.gz](https://github.com/raiden-network/light-client/files/5750346/bf4_node0_node3_29Dec2020.tar.gz)


BF6

[scenario-player-run_bf6_stress_hub_node_2020-12-29T00:54:52.log](https://github.com/raiden-network/light-client/files/5750351/scenario-player-run_bf6_stress_hub_node_2020-12-29T00.54.52.log)
[bf6_node1_node2_29thDec2020.tar.gz](https://github.com/raiden-network/light-client/files/5750355/bf6_node1_node2_29thDec2020.tar.gz)

**Comment by christianbrb** [comment]
Should get fixed by #2453
